src/track/heaptrack.sh.cmake

```diff
@@ -250,17 +258,27 @@ else
         --eval-command="run" --args "$client" "$@"
   else
     echo "injecting heaptrack into application via GDB, this might take some time..."
+    case $(uname) in
+        Linux*)
+            dlopen=__libc_dlopen_mode
+        ;;
+        FreeBSD*)
+            dlopen=dlopen@plt
+        ;;
+    esac
     if [ -z "$debug" ]; then
         gdb --batch-silent -n -iex="set auto-solib-add off" -p $pid \
             --eval-command="sharedlibrary libc.so" \
-            --eval-command="call (void) __libc_dlopen_mode(\"$LIBHEAPTRACK_INJECT\", 0x80000000 | 0x002)" \
+            --eval-command="set \$dlopen = '$dlopen'" \
```

Comment:
the set isn't needed, you could just pass expand the bash var directly in the line below by removing the `\\` before the `$` no?

Reply:
I did that to make it parse `dlopen@plt` correctly, but seems like I could just put it in quotes like `call 'dlopen@plt'()`

Reply:
yep that sounds better to me